### ISSUE TITLE
test_benchdnn_self_ci_cpu check_graph fails due to rdiff from jit:sve_256 eltwise

### ISSUE THREAD

**Issue by jondea** [oneapi-src/oneDNN#1533]
# Summary
`test_benchdnn_self_ci_cpu` is failing in `check_graph()` due to
```
onednn_verbose,create:cache_miss,cpu,eltwise,jit:sve_256,forward_training,data_f32::blocked:abcd:f0 diff_undef::undef::f0,,alg:eltwise_exp alpha:0 beta:0,2x128x3x3
```
which causes
```
[ 914][0:101:1:2] exp_f32:     1097.88 exp:     1097.88 got:     1097.88 diff:0.000244141 rdiff:2.22374e-07
```
The first commit to show this failure is 0185dcf33f872da24195080d1eb951cef1793a4c

I'm not sure what the fix is for this, but two potential ideas are:
- The error is only 2 ULP, so we could relax the tolerance (currently `1.2e-7`). The standalone eltwise benchdnn driver allows rdiffs of `4e-6` (https://github.com/oneapi-src/oneDNN/blob/0185dcf33f872da24195080d1eb951cef1793a4c/tests/benchdnn/eltwise/eltwise.cpp#L158-L168)
- Add a class flag for `has_exp_eltwise` like `has_eltwise_post_op_` so that this check is triggered when the graph driver is used https://github.com/oneapi-src/oneDNN/blob/0185dcf33f872da24195080d1eb951cef1793a4c/tests/benchdnn/utils/compare.cpp#L217-L223

# Version
Present on master d49bfbe6c2c0725902b43b3d15c72cb3303be55d, introduced in 0185dcf33f872da24195080d1eb951cef1793a4c

# Environment
* CPU make and model: AWS c7g
* OS version: Ubuntu 20.04
* Compiler version: fails with gcc-10 and clang-9, probably on others too
* CMake version: cmake version 3.16.3
* CMake output log: see below
* git hash: 0185dcf33f872da24195080d1eb951cef1793a4c

# Steps to reproduce
```
cmake .. -DCMAKE_BUILD_TYPE=Debug  
make
./tests/benchdnn/benchdnn -v1 --self --batch=test_self_ci
```

# Benchdnn output
```
check_simple_enums() ...
check_attr2str() ...
check_attr() ...
check_post_ops2str() ...
check_str2post_ops() ...
check_tags() ...
check_trim_tags() ...
check_skip_impl() ...
check_status_change() ...
No trust stats []: z:100% (>30%) (z: 10, total: 10)
[   0][0] exp_f32:           0 exp:           0 got:          -1 diff:       1 rdiff:       1
[   1][1] exp_f32:           1 exp:           1 got:           0 diff:       1 rdiff:       1
[   2][2] exp_f32:           2 exp:           2 got:           1 diff:       1 rdiff:     0.5
[   3][3] exp_f32:           3 exp:           3 got:           2 diff:       1 rdiff:0.333333
[   4][4] exp_f32:           4 exp:           4 got:           3 diff:       1 rdiff:    0.25
[   5][5] exp_f32:           5 exp:           5 got:           4 diff:       1 rdiff:     0.2
[   6][6] exp_f32:           6 exp:           6 got:           5 diff:       1 rdiff:0.166667
[   7][7] exp_f32:           7 exp:           7 got:           6 diff:       1 rdiff:0.142857
[   8][8] exp_f32:           8 exp:           8 got:           7 diff:       1 rdiff:   0.125
[   0][0] exp_f32:           0 exp:           0 got:          -1 diff:       1 rdiff:       1
[   1][1] exp_f32:           1 exp:           1 got:           0 diff:       1 rdiff:       1
[   2][2] exp_f32:           2 exp:           2 got:           1 diff:       1 rdiff:     0.5
[   3][3] exp_f32:           3 exp:           3 got:           2 diff:       1 rdiff:0.333333
[   4][4] exp_f32:           4 exp:           4 got:           3 diff:       1 rdiff:    0.25
[   5][5] exp_f32:           5 exp:           5 got:           4 diff:       1 rdiff:     0.2
[   6][6] exp_f32:           6 exp:           6 got:           5 diff:       1 rdiff:0.166667
[   7][7] exp_f32:           7 exp:           7 got:           6 diff:       1 rdiff:0.142857
[   8][8] exp_f32:           8 exp:           8 got:           7 diff:       1 rdiff:   0.125
check_simple_enums() ...
check_flags() ...
check_desc() ...
check_bool_operator() ...
check_graph() ...
[ 914][0:101:1:2] exp_f32:     1097.88 exp:     1097.88 got:     1097.88 diff:0.000244141 rdiff:2.22374e-07
@@@ error [int self::check_correctness(std::unordered_map<int, std::tuple<benchdnn_dnnl_wrapper_t<dnnl_primitive*>, std::unordered_map<int, dnn_mem_t, std::hash<int>, std::equal_to<int>, std::allocator<std::pair<const int, dnn_mem_t> > >, std::unordered_map<int, dnn_mem_t, std::hash<int>, std::equal_to<int>, std::allocator<std::pair<const int, dnn_mem_t> > >, args_t, args_t,
compare::compare_t> >&, const std::vector<data_kind_t>&, res_t*):219]: 'cmp.compare(mem_fp_golden, mem_dt, attr_t(), res)' ->
1
@@@ error [int self::check_graph():238]: 'check_correctness(op_graph, {DST}, res)' -> 1
tests:14 passed:13 skipped:0 mistrusted:0 unimplemented:0 invalid_arguments:0 failed:1 listed:0
total compute_ref: sum(s):0.00
```

# CMake output
```
-- The C compiler identification is GNU 10.3.0
-- The CXX compiler identification is GNU 10.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- DNNL_TARGET_ARCH: AARCH64
-- DNNL_LIBRARY_NAME: dnnl
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Found OpenMP_C: -fopenmp (found version "4.5")
-- Found OpenMP_CXX: -fopenmp (found version "4.5")
-- Found OpenMP: TRUE (found version "4.5")
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- Could NOT find Doxyrest (missing: DOXYREST_EXECUTABLE)
-- Found PythonInterp: /usr/bin/python2.7 (found suitable version "2.7.18", minimum required is "2.7")
-- Could NOT find Sphinx (missing: SPHINX_EXECUTABLE)
-- Found Git: /usr/bin/git (found version "2.25.1")
-- Enabled workload: TRAINING
-- Enabled primitives: ALL
-- Enabled primitive CPU ISA: ALL
-- Enabled primitive GPU ISA: ALL
-- Primitive cache is enabled
-- Configuring done
-- Generating done
-- Build files have been written to: oneDNN/build
```

**Comment by vpirogov** [comment]
+@dzarukin

**Comment by dzarukin** [comment]
I replaced exponent with relu. Once the change hits the master, the issue should be gone.

**Comment by nSircombe** [comment]
Thanks @dzarukin  - which commit carries the fix? We're skipping the test at present, but I'll re-enable once the change filters into our builds.

**Comment by vpirogov** [comment]
Fixed by b44df87ebed2be20b43000464c72f8f5d998f0bf
